src/calendar/index.tsx

```diff
@@ -387,7 +387,14 @@ export class Calendar extends I18nMixin(ThemedMixin(WidgetBase))<CalendarPropert
 		const labelText = type === Paging.next ? labels.nextMonth : labels.previousMonth;
 
 		return [
-			w(Icon, { type: iconType, theme, classes }),
+			w(Icon, {
+				type: iconType,
+				theme,
+				classes: {
```

Comment:
I believe this should use the `theme.composes` with a prefix otherwise this will clash with passed in classes.

Reply:
The calendar and combobox widget are not build using a widget factory and can't use `theme.compose`